full_data.json

```diff
@@ -15539,5 +15539,162 @@
             "character": "",
             "campaign": "standard"
         }
+    ],
+    "score_bonuses": [
```

Comment:
I kinda added this before I knew how it worked, this isn't actually needed, so if you want me to remove this from this file, I can.